rpm_spec/subpackages/manageiq-gemset

```diff
@@ -59,4 +59,5 @@ Requires: network-scripts
 %{gemset_root}/specifications
 %{gemset_root}/vmdb
 %{gemset_root}/enable
+%{_prefix}/lib/systemd/system/manageiq-providers*
```

Comment:
This is much nicer :+1:  If any plugins bring workers that _aren't_ providers...we'll have to deal with that somehow, but for now this is good.